frontend/src/citizen-frontend/applications/ChildApplicationsBlock.tsx

```diff
@@ -246,7 +247,8 @@ export default React.memo(function ChildApplicationsBlock({
                   {applicationStatus === 'WAITING_CONFIRMATION' &&
                     permittedActions[applicationId]?.includes(
                       'READ_DECISIONS'
-                    ) && (
+                    ) &&
+                    ownedByCurrentUser && (
```

Comment:
should this be also based on the canDecide or something more generic like that instead of kind of hard coding it here that the application owner can do this? 🤔

Reply:
It's a bit tricky since we have here data and permissions about applications but accepting/rejecting is an action done on the individual decisions (Action.Citizen.Decision). But I'll think about it a bit more...